src/SourceTerms/CPU_SrcSolver_IterateAllCells.cpp

```diff
@@ -48,7 +48,7 @@ void CUSRC_SrcSolver_IterateAllCells(
    const double g_Corner_Array[][3],
    const SrcTerms_t SrcTerms, const int NPatchGroup, const real dt, const real dh,
    const double TimeNew, const double TimeOld,
-   const real MinDens, const real MinPres, const real MinEint, const EoS_t EoS )
+   const real MinDens, const real MinPres, const real MinEint, const long PassiveFloor, const EoS_t EoS )
```

Comment:
Please pass `PassiveFloor` to the source term functions `SrcTerms.Dlep_FuncPtr()` and `SrcTerms.User_FuncPtr()` at lines 113 and 119.

You will also need to update the prototype in `include/SrcTerms.h`, and `Src_User_Template()` in `src/SourceTerms/User_Template` (same for `Deleptonization`).